purchase_line_product_image/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
```

Comment:
@kurniawanlucky License says LGPL here but the files headers and manifest says AGPL.

Reply:
@max3903 yes, it's done
thanks for your review